src/incubator/WheelPicker/index.tsx

```diff
@@ -39,47 +46,104 @@ export interface WheelPickerProps {
   /**
    * Event, on active row change
    */
-  onChange: (index: number, item?: ItemProps) => void;
```

Comment:
Don't you prefer to keep it synced with the Picker's `onChange` API?

Reply:
Do you mean to `onChange: PropTypes.func` without specified values / index?